src/index.js

```diff
@@ -142,10 +151,37 @@ function init( {
 					} else {
 						node.addEventListener( 'mouseenter', showPopup )
 					}
+
+					foundDetectLinks.push( {
+						text: node.textContent,
+						title: matches.title,
+						lang: matches.lang
+					} )
 				}
 			}
 		)
 	}
+
+	if ( debug ) {
+		/* eslint-disable no-console */
+		console.group( 'Wikipedia Preview [debugger]' )
+		console.log( `searching for the content in language "${globalLang}" within the selector of "${selector}"` )
+		console.group( `Total links found in the given selector of "${selector}" : ${foundSelectorLinks.length}` )
+		foundSelectorLinks.forEach( ( link, index ) => {
+			console.log( index + 1, `text "${link.text}", title "${link.title}", lang : "${link.lang}"` )
+		} )
+		console.groupEnd()
+		if ( detectLinks ) {
+			console.group( `Total links found in the hyperlink : ${foundDetectLinks.length}` )
```

Comment:
Suggestion: Searching for links to Wikipedia... Found: ${foundDetectLinks.length}